crates/api/src/types.rs

```diff
@@ -435,27 +412,9 @@ impl ImportType {
 /// exported.
 #[derive(Debug, Clone, Hash, Eq, PartialEq)]
 pub struct ExportType {
-    name: String,
-    ty: ExternType,
-}
-
-impl ExportType {
-    /// Creates a new export which is exported with the given `name` and has the
-    /// given `ty`.
-    pub fn new(name: &str, ty: ExternType) -> ExportType {
-        ExportType {
-            name: name.to_string(),
-            ty,
-        }
-    }
+    /// The name of the export.
+    pub name: String,
```

Comment:
Similar to above, I think we should keep this as an internal field with a public accessor so we can change the make up of the structure over time.

Reply:
Also fixed.